crates/node/Cargo.toml

```diff
@@ -19,11 +19,7 @@ reth-chainspec.workspace = true
 eyre.workspace = true
 
 [features]
-default = [
-  "alphanet-precompile/optimism",
-  "reth/optimism",
-  "reth-node-optimism/optimism",
-]
+default = ["reth-node-optimism/optimism"]
```

Comment:
do we even need these as default features, can't this be enabled on the crate dep directly?
or on workspace level?

Reply:
yes

Reply:
i'd prefer to do this in a separate pr since this is unrelated